### ISSUE TITLE
Navigation with Alt+Left/Right doesn't work with grace notes and sub note in bends  (Screen Readers)

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#19926]
### Issue type

UI bug

### Bug description

Acciaccatura's are read by a screen reader when navigating along with arrows, but are skipped over when navigating all elements using Alt arrows.

Occurs with **all grace notes** and **sub note** in **bends**

### Steps to reproduce

1.  Insert an acciaccatura into a score.
2. Arrow across it and you will hear the acciaccatura read first and then with another right arrow press the note it is attached to.
3. Repeat this using Alt + right arrow starting from before the acciaccatur.
4.  the acciaccatura is skipped over and not read or selected.

### Screenshots/Screen recordings

[see comment](https://github.com/musescore/MuseScore/issues/19926#issuecomment-1804061216)

### MuseScore Version

4.1

### Regression

I don't know

### Operating system

Windows 10 with NVDA

**Comment by DmitryArefiev** [comment]
This doesn't work with Navigation initially.

Occurs with **all grace notes** and **sub note** (appoggiatura) **in bends** . I'll rewrite the summary


https://github.com/musescore/MuseScore/assets/90187801/8004154d-f5e3-414c-932a-9e8db585c276

**Comment by jmwmusic** [comment]
When navigating with Alt + arrow keys it is now possible to move to the grace note.  However, all other elements such as articulation, lyrics, etc are skipped over when moving from the grace note or its main note to the next or previous notes.  Arrowing from the next or previous note towards the grace note does reveal all the elements in between.
